src/FSharpLint.Core/Rules/Conventions/NumberOfItems/MaxNumberOfMembers.fs

```diff
@@ -25,13 +25,18 @@ let private validateType (maxMembers:int) members typeRepresentation =
     let members =
         match typeRepresentation with
         | SynTypeDefnRepr.Simple(_) | SynTypeDefnRepr.Exception(_) -> members
-        | SynTypeDefnRepr.ObjectModel(_, members, _) -> members
-        |> getMembers
+        | SynTypeDefnRepr.ObjectModel(_, members, _) -> getMembers members
 
     if List.length members > maxMembers then
         let errorFormatString = Resources.GetString("RulesNumberOfItemsClassMembersError")
         let error = String.Format(errorFormatString, maxMembers)
-        { Range = members.[maxMembers].Range; Message = error; SuggestedFix = None; TypeChecks = [] } |> Array.singleton
+        {
+            Range = members.[maxMembers].Range
+            Message = error
+            SuggestedFix = None
+            TypeChecks = List.Empty
+        }
+        |> Array.singleton
```

Comment:
@Mersho is this a false negative of AvoidSinglePipeOperator?

Reply:
> @Mersho is this a false negative of AvoidSinglePipeOperator?

yes, i can create a PR and address it if you want.

Reply:
yes please